src/main/resources/org/jenkinsci/plugins/awsdevicefarm/AWSDeviceFarmRecorder/config.jelly

```diff
@@ -61,6 +65,10 @@
         <f:entry title="Tests" field="appiumJavaJUnitTest" description="[Required] Pattern to find Java JUnit tests.">
           <f:textbox />
         </f:entry>
+
+        <f:entry name="appiumVersion" title="Appium Version" field="appiumVersionJunit">
+          <f:select style="width:100%; padding-right:5px; margin-right:5px" inline="true" />
```

Comment:
Highly recommend using the actual CSS file instead of inline styles.

Single note, but there are multiple places where you're doing this and just copy pasting the values over and over.